src/main/java/tectech/thing/metaTileEntity/multi/MTEEyeOfHarmony.java

```diff
@@ -837,9 +836,9 @@ private int recipeProcessTimeCalculator(final long recipeTime, final long recipe
         // = 3%*3% = 5.91% discount.
 
         final long spacetimeCasingDifference = (recipeSpacetimeCasingRequired - spacetimeCompressionFieldMetadata);
-        final double recipeTimeDiscounted = recipeTime * pow(2.0, -timeAccelerationFieldMetadata)
-            * pow(1 - SPACETIME_CASING_DIFFERENCE_DISCOUNT_PERCENTAGE, -spacetimeCasingDifference)
-            / max(1, pow(2, currentCircuitMultiplier));
+        final double recipeTimeDiscounted = recipeTime * GTUtility.powInt(2.0, -timeAccelerationFieldMetadata)
+            * GTUtility.powInt(1 - SPACETIME_CASING_DIFFERENCE_DISCOUNT_PERCENTAGE, -spacetimeCasingDifference)
+            / max(1, GTUtility.powInt(2, currentCircuitMultiplier));
```

Comment:
Not you, but there's no reason to have this division instead of a third negative power
```suggestion
        final double recipeTimeDiscounted = recipeTime * GTUtility.powInt(2.0, -timeAccelerationFieldMetadata)
            * GTUtility.powInt(1 - SPACETIME_CASING_DIFFERENCE_DISCOUNT_PERCENTAGE, -spacetimeCasingDifference)
            * min(1, GTUtility.powInt(2, -currentCircuitMultiplier));
```